ocpp/v16/charge_point.py

```diff
@@ -205,7 +205,10 @@ def __init__(self, id, connection, response_timeout=30):
 
         try:
             handler = handlers['_after_action']
-            await asyncio.coroutine(handler)(**snake_case_payload)
+            # Create task to avoid blocking when making a call inside the
+            # after handler
+            asyncio.create_task(
```

Comment:
This fails in python3.6. instead you can use asyncio.ensure_future(coro()), which works in all python versions

Reply:
check  the doc of asyncio :
[https://docs.python.org/3/library/asyncio-task.html#id4](url)

Reply:
Ughhh.